### PR TITLE
 Lifecycle Hooks

### PR DESCRIPTION
**Pull request by KritiG7**
<hr>

## Description 📜

Fixes issue #6903 Lifecycle Hooks 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X ] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ X] I have performed a self-review of my own code or work.
- [X ] I have commented my code, particularly in hard-to-understand areas.
- [ X] My changes generates no new warnings.
- [ X] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by KritiG7** [comment]
Closes #6903

File: Frontend_Web_Development_React_Angular_Vue/Vue/Week 6/Lifecycle Hooks.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  },
+  unmounted() {
+    console.log('Parent unmounted()')
+  },
+}
+</script>
+
+<style scoped>
+</style>
+```
```
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/Vue/Week 6/Lifecycle Hooks.md:89]: made this code snippets of the javascript or jsx

File: Frontend_Web_Development_React_Angular_Vue/Vue/Week 6/Lifecycle Hooks.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  },
+  unmounted() {
+    console.log('Child unmounted()')
+  },
+}
+</script>
+
+<style scoped>
+</style>
+```
```
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/Vue/Week 6/Lifecycle Hooks.md:128]: make this code snippets of the javaScript or jsx to.....

**Review by heet-kumar — CHANGES_REQUESTED** [review]
@KritiG7 that looks amazing just add in the starting after the 3 back ticks (```jsx) write jsx.

**Comment by SourabhSaraswat-191939** [comment]
Hey @KritiG7, your PR looks good. But you had not used the life cycle diagram. It will be better for beginners to understand it with a diagram.
If you want to copy diagram images from any source then don't forget to mention their name in resources or near the image.

After this change we will merge your PR.

**Comment by KritiG7** [comment]
@SourabhSaraswat-191939 I have made the required changes.Kindly merge it.

**Comment by SourabhSaraswat-191939** [comment]
@anandhu720 , please merge this PR.

**Review by anandhu720 — APPROVED** [review]

**Pull request merged**